src/services/ui/src/components/Layout/index.tsx

```diff
@@ -19,6 +19,10 @@ const getLinks = (isAuthenticated: boolean) => {
         name: "Dashboard",
         link: "/dashboard",
       },
+      {
+        name: "FAQ",
+        link: "/faq",
+      },
```

Comment:
this is great but you also need to add it to the router.ts file so the page exists in the routing. im honestly not even sure how you are developing this without seeing it lol.